.github/workflows/prepare-for-prod-deploy.yml

```diff
@@ -92,6 +92,9 @@ jobs:
           echo "branch_name=$branch_name"
           echo "branch_name=$branch_name" >> $GITHUB_OUTPUT
 
+      - name: Install hub
```

Comment:
Instead of installing the missing `hub`, use the official GitHub CLI `gh`, which should be already part of the runners.
https://github.com/cli/cli/blob/trunk/docs/gh-vs-hub.md

Reply:
replaced hub with `gh`

Reply:
test Pr created automatically - https://github.com/rudderlabs/rudder-devops/pull/8854